packages/lib-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@openshift/dynamic-plugin-sdk-utils",
-  "version": "4.1.0",
+  "version": "5.0.0",
```

Comment:
@vojtechszocs While none of the UI components being removed seem to be used in production environments as far as we can tell - I am proposing we bump a major version of the lib-utils package in case we missed any locations.

Reply:
Sounds good, we can also publish that new version to npmjs repository once the PR gets merged.